frontend/workflows/projectSelector/src/project-group.tsx

```diff
@@ -226,6 +226,7 @@ const ProjectGroup: React.FC<ProjectGroupProps> = ({ title, group, displayToggle
                           payload: { group, projects: [key] },
                         })
                       }
+                      size="large"
```

Comment:
Did the default value change here? Why change this to a large size?

Reply:
Yup, new default should now be "large" due to a padding change:

https://mui.com/material-ui/migration/v5-component-changes/#%E2%9C%85-update-size-prop